tests/unit/specs/game-state.spec.js

```diff
@@ -1,34 +1,49 @@
-import { describe, it , expect, beforeEach} from 'vitest';
-import { fixture } from '../fixtures/gameState';
+import { describe, it, expect, beforeEach } from 'vitest';
+import { fixture, resolveThree } from '../fixtures/gameState';
 
-describe('Coverting GameState\'s and GameStateRow\'s', () => {
+//remove attributes added while creating the entry in the database
+const stripDbAttributes = (obj) => {
+  const attributesToRemove = ['createdAt', 'id', 'updatedAt'];
+  attributesToRemove.forEach((attr) => delete obj[attr]);
+};
 
+describe("Coverting GameState's and GameStateRow's", () => {
   beforeEach(async () => {
     await sails.helpers.wipeDatabase();
   });
 
   it('Converts GameState to GameStateRow (packing)', async () => {
     const gameStateRow = await sails.helpers.gamestate.saveGamestate(fixture.gameState);
 
-    //remove attributes added while creating the entry in the database
-    const stripDbAttributes = (obj) => {
-      const attributesToRemove = ['createdAt', 'id', 'updatedAt'];
-      attributesToRemove.forEach(attr => delete obj[attr]);
-    };
-
     stripDbAttributes(gameStateRow);
     expect(gameStateRow).toEqual(fixture.gameStateRow);
 
-    const dbGameStateRows = await GameStateRow.find({ gameId : gameStateRow.gameId });
+    const dbGameStateRows = await GameStateRow.find({ gameId: gameStateRow.gameId });
     expect(dbGameStateRows.length).toEqual(1);
 
-    const [ savedRow ] = dbGameStateRows;
+    const [savedRow] = dbGameStateRows;
     stripDbAttributes(savedRow);
     expect(savedRow).toEqual(fixture.gameStateRow);
   });
 
   it('Converts GameStateRow to GameState (unpacking)', () => {
-    const gameState =  sails.helpers.gamestate.unpackGamestate(fixture.gameStateRow);
+    const gameState = sails.helpers.gamestate.unpackGamestate(fixture.gameStateRow);
     expect(fixture.gameState).toEqual(gameState);
   });
 });
+
+describe('resolveThree', () => {
```

Comment:
On the organization of describe() and it():

I think we should either have one describe block per fixture with separate tests for each function+assertion, or put them all in one describe() block containing every fixture, with one it() per fixture. There's no need for separate describe blocks if they each only contain one test.

Which would you prefer?